client/interledger/js/components/assets.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import AssetDetail from '../../../lib/js/react/components/asset_detail';
+
+
+export default function Assets(props) {
```

Comment:
Put the `export` declaration at the end of the file, after the `propTypes` have been attached.